.build

```diff
@@ -1,4 +1,5 @@
 CC = gcc
```

Comment:
it's not my tool, that's why I created a new org (which is eagerly awaiting you to join) and invited you all.
the .build file comes from the buck and bazel family, I really didn't think too much over it. I'll create an issue and change the .build to something like `bldyconfig`